cmd/acra-connector/acra-connector.go

```diff
@@ -149,23 +149,23 @@ func main() {
 	keysDir := flag.String("keys_dir", keystore.DEFAULT_KEY_DIR_SHORT, "Folder from which will be loaded keys")
 	clientId := flag.String("client_id", "", "Client id")
 	acraHost := flag.String("acra_host", "", "IP or domain to acra daemon")
-	acraCommandsPort := flag.Int("acra_commands_port", cmd.DEFAULT_ACRA_API_PORT, "Port of acra http api")
+	acraCommandsPort := flag.Int("acra_commands_port", cmd.DEFAULT_ACRA_API_PORT, "Port of Acra HTTP api")
```

Comment:
Here and further we use the term `Acra port` for different instances: AcraServer and AcraConnector. It would be good to explicitly specify application. In this line, I think, should be AcraServer.